CHANGELOG.md

```diff
@@ -49,6 +49,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
     if no exact binding (i.e. one with `Shift`) is found.
 - Use built-in font for powerline symbols from `U+E0B0` to `U+E0B3`
 - Default `bell.animation` is now `Linear`
+- `{Increase,Decrease}FontSize` step is now 1px
+- `font.size` precision was raised to 6 fp digits
```

Comment:
What's a user supposed to do with "6 fp digits"? At least write it as "floating point" so you don't have to guess…